src/phasorpy/phasor.py

```diff
@@ -0,0 +1,511 @@
+"""Calculate, convert, calibrate, and reduce phasor coordinates.
+
+The ``phasorpy.phasor`` module provides functions to:
+
+- calculate phasor coordinates from time-resolved and spectral signals:
+
+  - :py:func:`phasor_from_signal` (not implemented yet)
+
+- calculate phasor coordinates from single- or multi-component fluorescence
+  lifetimes:
+
+  - :py:func:`phasor_from_lifetime`
+  - :py:func:`phasor_from_apparent_lifetime` (not implemented yet)
+  - :py:func:`phasor_to_apparent_lifetime` (not implemented yet)
+
+- convert between phasor and polar (phase and modulation) coordinates:
+
+  - :py:func:`phasor_from_polar` (not implemented yet)
+  - :py:func:`phasor_to_polar`
+
+- calibrate phasor coordinates with reference of known fluorescence
+  lifetime:
+
+  - :py:func:`phasor_calibrate`
+  - :py:func:`polar_from_reference`
+  - :py:func:`polar_from_reference_phasor`
+
+- reduce arrays of phasor coordinates to single coordinates:
+
+  - :py:func:`phasor_center`
+
+"""
+
+from __future__ import annotations
+
+__all__ = [
+    "phasor_calibrate",
+    "polar_from_reference_phasor",
+    "polar_from_reference",
+    "phasor_to_polar",
+    "phasor_from_lifetime",
+    "phasor_center",
+]
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, NDArray, ArrayLike, Literal
+
+import math
+
+import numpy
+
+
+def phasor_calibrate(
+    real: ArrayLike,
+    imag: ArrayLike,
+    phase0: ArrayLike = 0.0,
+    modulation0: ArrayLike = 1.0,
+    /,
+) -> tuple[NDArray[Any], NDArray[Any]]:
+    """Return calibrated/referenced phasor coordinates.
+
+    Parameters
+    ----------
+    real : array_like
+        Scalar or array containing the real components of phasor coordinates.
+    imag : array_like
+        Scalar or array containing the imaginary components of phasor
+        coordinates.
+    phase0 : array_like, optional
+        Angular component of polar coordinates for calibration. Defaults
+        to 0.0.
+    modulation0 : array_like, optional
+        Radial component of polar coordinates for calibration. Defaults to 1.0.
+
+    Raises
+    ------
+    ValueError
+        If the shapes of 'phase0' and 'modulation0' arrays do not match.
+        If the shapes of 'real' and 'imag' arrays do not match.
+
+    Returns
+    -------
+    real, imag:
+        Calibrated real and imaginary components of phasor coordinates.
+
+    Examples
+    --------
+    >>> real = numpy.array([1.0, 2.0, 3.0])
+    >>> imag = numpy.array([4.0, 5.0, 6.0])
+    >>> phase0 = 0.5
+    >>> modulation0 = 2.0
+    >>> real_calibrated, imag_calibrated = phasor_calibrate(
+    ...     real, imag,
+    ...     phase0,
+    ...     modulation0
+    ... )
+    >>> real_calibrated
+    array(...)
+    >>> imag_calibrated
+    array(...)
```

Comment:
I find these examples quite verbose for docstrings. The function signature is explained just a few lines before and these docstrings are often displayed by IDEs in small  popup windows.

How about:
```
Examples
--------
Use scalar reference coordinates to calibrate phasor coordinates:
>>> phasor_calibrate([1.0, 2.0, 3.0], [4.0, 5.0, 6.0], 0.5, 0.2)
(array(...), array(...))

Use separate reference coordinates for each phasor coordinate:
>>> phasor_calibrate([1.0, 2.0, 3.0], [4.0, 5.0, 6.0], [0.5, 0.2, 0.3], [1.5, 2.0, 0.3])
(array(...), array(...))
```
What do you think? Maybe get some feedback from Bruno and Leonel?

Reply:
Bruno and Leonel also agreed that maybe the less verbose and more concise example are better. So we can start using that criteria from now on.

Reply:
Good. We can always improve/revise/add later.